README.md

```diff
@@ -1,264 +1,483 @@
-
-`CONJUR_APPLIANCE_URL` - Conjur HTTPS endpoint (OSS/DAP)
 
-`CONJUR_AUTHN_LOGIN` - user/host identity
-
-`CONJUR_AUTHN_API_KEY` - user/host API key
+Once the setup steps have been successfully run, we will now define the variables needed
+to make the connection between the new app and Conjur. You can do this by setting
+[environment variables](#environment-variables), [system properties](#system-properties),
+or some combination of both.
+
+_NOTE:_ System properties will override enviroment values when both are defined for a
+variable.
+
+### Environment Variables
+
+In Conjur (both OSS and DAP), environment variables are mapped to configuration variables
+by prepending `CONJUR_` to the all-caps name of the configuration variable. For example,
+`appliance_url` is `CONJUR_APPLIANCE_URL`, `account` is `CONJUR_ACCOUNT` etc.
+
+_NOTE:_ For ways to set credentials (for `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY`
+environment variables ONLY) in the app instead of environment variables, see the
+[Authorization Examples](#authorization-examples) and/or [system properties](#system-properties)
+section below.
+
+The following environment variables need to be included in the apps runtime environment in
+order use the Conjur API:
+
+- `CONJUR_APPLIANCE_URL` - The URL of the Conjur instance you are connecting to. When connecting to
+  DAP configured for high availability, this should be the URL of the master load balancer (if
+  performing read and write operations) or the URL of a follower load balancer (if performing
+  read-only operations).
+- `CONJUR_ACCOUNT` - Conjur account that you are connecting to. This value is set during Conjur deployment.
+- `CONJUR_AUTHN_LOGIN` - User/host identity
+- `CONJUR_AUTHN_API_KEY` - User/host API key
+- `CONJUR_AUTHN_URL` - (optional) Alternate authentication endpoint. By default the client
+  uses the standard `<applianceUrl>/authn` for generic username and API key login flow.
+
+For example, you can specify the environment variables like so:
+
+```sh-session
+export CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api
+export CONJUR_ACCOUNT=myorg
+export CONJUR_AUTHN_LOGIN=host/myhost.example.com
+export CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
+```
 
-`CONJUR_AUTHN_URL` - (optional) Set this variable if using alternative authentication method
+or you could provide these at runtime to your jar:
+```sh-session
+$ CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api \
+  CONJUR_ACCOUNT=myorg \
+  CONJUR_AUTHN_LOGIN=host/myhost.example.com \
+  CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s \
+  java -jar myConjurClient.jar
+```
+If you are using a host-based user like this example shows, you will need to add the host
+to Conjur with the proper privileges in policy in order to know the appropriate
+`CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY` values.
+
+### System Properties
+
+This API can also be configured using [Java system properties](https://www.baeldung.com/java-system-get-property-vs-system-getenv).
```

Comment:
Maybe better to link to official docs? https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html